source/sass/main.scss

```diff
@@ -38,6 +38,9 @@
 @import "./wagtail/blocks/dear-internet-letter";
 @import "./wagtail/blocks/feature-quote";
 @import "./wagtail/blocks/image-text-mini";
+@import "./wagtail/blocks/paragraph";
```

Comment:
let's rename this one to `rich-text`, as the block gives folks a WYSIWYG editor that lets them insert far more than just paragraphs, but the only custom styling we needed was around the way paragraphs got spaced.